vault/identity_store_identities.go

```diff
@@ -0,0 +1,342 @@
+package vault
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/fatih/structs"
+	memdb "github.com/hashicorp/go-memdb"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+// identityPaths returns the API endpoints to operate on identities.
+// Following are the paths supported:
+// identity - To register/modify an identity
+// identity/id - To lookup, delete and list identities based on ID
+func identityPaths(i *identityStore) []*framework.Path {
+	return []*framework.Path{
+		{
+			Pattern: "identity",
```

Comment:
Discussed offline and `/v1/identity/persona` seems to solve the ambiguity between feature name "identity" and the "identity" within entities.